client/gogeta_client.go

```diff
@@ -41,39 +55,121 @@ func (client *GogetaClient) Start() {
 	store.Compression = zipCompress
 	store.StorageSystem = cloudStorage
 
-	client.Queue = queuesystem.AmazonQueue{
+	// queue system
+	sess := session.Must(session.NewSession())
+	amazonQueue := queuesystem.AmazonQueue{
 		Client: sqs.New(sess),
-		Region: os.Getenv(config.Region),
 		URL:    os.Getenv(config.QueueURL),
 	}
+
+	// publisher system
+	amazonSNS := publisher.AmazonNotification{}
```

Comment:
it essentially is, I haven't made an abstract client thing yet, I've just glued the pieces on top. For now we'll only have the gogeta client but in the future I can make this more general.